### ISSUE TITLE
[QUESTION] Issue with permissions on CSI powerscale mounts in openshift 4.14

### ISSUE THREAD

**Issue by mikelaff-anf** [dell/csm#1420]
### How can the Team help you today?

*We're having an issue with CSI powerscale.
In the values file, we define the permissions (with the isiVolumePathPermissions parameter inside a values.yml file in our ansible playbook) on a volume mountpoint as 0777.
However, when it mounts inside the pod, it mounts with 0770 and this is causing issues for us.
Has anyone else encountered this issue? Is setting the permissions in this way not supported?
We are running openshift 4.14
currently v2.10 is our CSI Powerscale version

**Comment by cflex** [comment]
This issue has been fixed. We discovered on the Powerscale side the parent folder the CSI driver was creating folders in had an extended ACL with inheritance set which was taking precedence over the permission the CSI driver was trying to set on the folder via the isiVolumePathPermissions parameter. Once we removed the extended ACL and added permission for the user the CSI driver is running under to be able to create folders this resolved our issue. 

`chmod -b 770 [folder]
chmod +a user [csiUser] allow dir_gen_read,dir_gen_write,dir_gen_execute,std_write_dac,delete_child [folder]`
